src/qt/peertablemodel.h

```diff
@@ -60,11 +60,12 @@ class PeerTableModel : public QAbstractTableModel
     enum ColumnIndex {
         NetNodeId = 0,
         Address = 1,
-        Network = 2,
-        Ping = 3,
-        Sent = 4,
-        Received = 5,
-        Subversion = 6
+        Direction = 2,
+        Network = 3,
+        Ping = 4,
+        Sent = 5,
+        Received = 6,
+        Subversion = 7
```

Comment:
To improve maintainability maybe explicitly assign a value to the first element only?

Reply:
done